README.md

```diff
@@ -264,13 +270,14 @@ ChatAction action = ChatAction.find_location;
 
 ### Available methods
 
-All request methods have the same names as original ones.  
-Required params should be passed in constructor.  
```

Comment:
Can we keep such double spaces please? They work like newlines

Reply:
Yes, I readded them in 12365024392d89c847a9117b2727b767366e46e5.